functions/functions/models/schemas.py

```
@@ -96,3 +96,48 @@ def _validate_redirect_uri(cls, uri: HttpUrl) -> HttpUrl:
     providerAccountId: str = Field(
         ..., description="ID of the provider account", min_length=1
     )
+
+
+class CreateNewTargetCalendarInput(BaseModel):
+    """Specifies details for creating a new target calendar during sync profile creation."""
+
+    type: Literal["createNew"]
+    colorId: int | None = Field(
```

Ruff (0.8.2)

Variable `colorId` in class scope should not be mixedCase

(N815)